cypress/Objects/LoginpageOpjects.js

```diff
@@ -0,0 +1,59 @@
+class LoginPage{
+    visit() {
+        cy.visit('/')
+    }
+
+    getUserValidationIndicator(){
+        return cy.get('.Select.not-valid')
+    }
+
+    getLoginButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getLoginFormUserId(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getUserName(userName){
+        return cy.get('[aria-label="'+ userName +'"]')
+    }
+
+    getLoginFormRoleId(){
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getUserRole(){
+        return  cy.get('[aria-label="Team Lead"]')
+    }
+
+    getSubmitButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getUrl(){
+        return cy.url().should('include', '/time-logging')
```

Comment:
You are not supposed to write asserts in this place, it's supposed to go to test file